src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -278,12 +276,6 @@ public void error(ExecutorDriver driver, String message) {
     log.error(this.getClass().getName() + ".error: " + message);
   }
 
```

Comment:
But I do shut down the driver more gracefully. I call killTask on the tasks. What else would you want to do in a graceful shutdown?